docs/getting-started/provider-aws.md

```diff
@@ -0,0 +1,219 @@
+Other authentication methods exist and are beyond the scope of this guide. The [Provider documentation](https://marketplace.upbound.io/providers/upbound/provider-aws/latest/docs/configuration) contains information on alternative authentication methods. 
+{{< /hint >}}
+
+#### Generate an AWS key-pair file
+For basic user authentication, use an AWS Access keys key-pair file. 
+
+{{< hint type="tip" >}}
+The [AWS documentation](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-creds) provides information on how to generate AWS Access keys.
+{{< /hint >}}
+
+Create a text file containing the AWS account `aws_access_key_id` and `aws_secret_access_key`.  
+
+```ini {copy-lines="all"}
+[default]
+aws_access_key_id = <aws_access_key>
+aws_secret_access_key = <aws_secret_key>
+```
+
+Save this text file as `aws-credentials.txt`.
+
+{{< hint type="note" >}}
+The [Configuration](https://marketplace.upbound.io/providers/upbound/provider-aws/latest/docs/configuration) section of the Provider documentation describes other authentication methods.
+{{< /hint >}}
+
+#### Create a Kubernetes secret with the AWS credentials
+A Kubernetes generic secret has a name and contents. Use {{< hover label="kube-create-secret" line="1">}}kubectl create secret{{< /hover >}} to generate the secret object named {{< hover label="kube-create-secret" line="2">}}aws-secret{{< /hover >}} in the {{< hover label="kube-create-secret" line="3">}}crossplane-system{{</ hover >}} namespace.  
+Use the {{< hover label="kube-create-secret" line="4">}}--from-file={{</hover>}} argument to set the value to the contents of the  {{< hover label="kube-create-secret" line="4">}}aws-credentials.txt{{< /hover >}} file.
+
+```shell {label="kube-create-secret",copy-lines="all"}
+kubectl create secret \
+generic aws-secret \
+-n crossplane-system \
+--from-file=creds=./aws-credentials.txt
+```
+
+View the secret with `kubectl describe secret`
+
+{{< hint type="note" >}}
+The size may be larger if there are extra blank spaces in your text file.
+{{< /hint >}}
+
+```shell
+kubectl describe secret aws-secret -n crossplane-system
+Name:         aws-secret
+Namespace:    crossplane-system
+Labels:       <none>
+Annotations:  <none>
+
+Type:  Opaque
+
+Data
+====
+creds:  114 bytes
+```
+
+### Create a ProviderConfig
+A `ProviderConfig` customizes the settings of the AWS Provider.  
+
+Apply the {{< hover label="providerconfig" line="2">}}ProviderConfig{{</ hover >}} with the command:
+```yaml {label="providerconfig",copy-lines="all"}
+cat <<EOF | kubectl apply -f -
+apiVersion: aws.upbound.io/v1beta1
+kind: ProviderConfig
+metadata:
+  name: default
+spec:
+  credentials:
+    source: Secret
+    secretRef:
+      namespace: crossplane-system
+      name: aws-secret
+      key: creds
+EOF
+```
+
+This attaches the AWS credentials, saved as a Kubernetes secret, as a {{< hover label="providerconfig" line="9">}}secretRef{{</ hover>}}.
```

Comment:
Is that good practice? Any viable alternative?

Reply:
No it's not good practice and the alternatives are more complex. The goal here is "easiest way to see Crossplane do something". There are two different callouts to alternative authentication methods, the first one is [here](https://github.com/crossplane/crossplane/pull/3396/files/91bfab79cfc02c7b1b9be09a49cf97a416820c77#diff-207d19c174999fbb3184ab046d25344852149f47c145e7b78f45aae74e84a2f5R83)

Reply:
Call outs have been added for all 3 guides.